### PR TITLE
[TRIVIAL] Publish new manifest signature

### PR DESCRIPTION
**Pull request by wilsonianb**
Minor changes related to https://github.com/ripple/rippled/pull/1865

### CODE REVIEW

File: src/ripple/app/misc/NetworkOPs.cpp [hunk 1]
```diff
@@ -2043,7 +2046,7 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin)
             if (! validation_manifest.lines().empty())
             {
                 std::string s;
-                s.reserve (188);
+                s.reserve (288);
```
Comment (scottschurr) [inline src/ripple/app/misc/NetworkOPs.cpp]: I'm not fond of the magic numbers here and in OverlayImpl.cpp, but I can't think of a better alternative.  And the cost if the number becomes wrong is relatively low.

**Comment by seelabs** [comment]
:+1: Nit: May want to re-word the commit message, we usually don't use tags like `[TRIVIAL]` in the actual commit message.

File: src/ripple/app/misc/NetworkOPs.cpp (hunk 1 above)
Comment (scottschurr) [inline src/ripple/app/misc/NetworkOPs.cpp]: That's a thought.  I usually reserve all caps for macros.  You might consider putting this declaration inside `struct Manifest`:
```
struct Manifest {
    ...
    static std::size_t constexpr textLength = 288;
    ...
```
Then you would replace the two `288`s with `Manifest::textLength`.

**Comment by wilsonianb** [comment]
Added `Manifest::textLength` and dropped `[TRIVIAL]`

**Comment by seelabs** [comment]
:+1:

**Comment by scottschurr** [comment]
:+1:

**Comment by bachase** [comment]
Merged in bbf52056f95f41a0d64227da3ea10c89d9cb3590

**Pull request closed without merging**